aws/assume-role/mint-leaf.yml

```diff
@@ -32,6 +32,11 @@ tasks:
     run: |
       set -ueo pipefail
 
+      cat << EOF > $(mktemp "$MINT_WARNINGS/warning-XXXX")
+      This leaf is deprecated. Please upgrade to \`2.0.0\` or later.
+      Going forward the OIDC token is provided by downstream tasks via AWS_OIDC_TOKEN environment variable.
```

Comment:
It'd be nice to link to the README for the new leaf version to provide a more thorough description of how to migrate.

Reply:
Good idea! Will do.